### PR TITLE
Add more test

### PR DESCRIPTION
**Pull request by Roger-luo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,15 @@ matrix:
...
 notifications:
   email: false
+jobs:
+  include:
+    - stage: "DocTest"
+      julia: 1.1
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+        - julia --project=docs/ docs/make.jl
```
Comment (fredrikekre) [inline .travis.yml]: Doesn't work if you add
```suggestion
        - julia --project=docs/ --code-coverage=user docs/make.jl
```
?

Comment (Roger-luo) [reply .travis.yml]: I added this here https://github.com/QuantumBFS/YaoBlocks.jl/pull/28/commits/7968d20483ff0029e75018e5eaa8e2b8f776ef11

Is it correct? but it should show the printing of some types are covered at least... and it didn't appear in codecov

Comment (fredrikekre) [reply .travis.yml]: Seems to have worked on this branch when we tested: https://github.com/JuliaDocs/Documenter.jl/compare/mp/coverage https://codecov.io/gh/JuliaDocs/Documenter.jl/branch/mp%2Fcoverage

Comment (Roger-luo) [reply .travis.yml]: Ok... let me try again.

**Comment by fredrikekre** [comment]
I think the problem might be that no doctests are run due to these errors: https://travis-ci.com/QuantumBFS/YaoBlocks.jl/jobs/196581924#L569 since `Yao` is not in `docs/Project.toml`.

File: .travis.yml [hunk 2]
```diff
@@ -13,6 +13,15 @@ matrix:
...
   email: false
+jobs:
+  include:
+    - stage: "DocTest"
+      julia: 1.1
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+        - julia --project=docs/ --code-coverage=user docs/make.jl
+      name: "HTML"
```
Comment (fredrikekre) [inline .travis.yml]: If you just remove this line this build stage will be called `DocTest` instead of `HTML`.

Edit: Nevermind, it already does, but it will be labeled based on Julia version instead of HTML.

Comment (Roger-luo) [reply .travis.yml]: Nice, I think it works now! I see. thanks for the help!
